frontend/src/assets/manual/en-GB/citation.md

```diff
@@ -0,0 +1,36 @@
+## Citing documents from I-analyzer
+
+If you want to cite a document that you accessed through I-analyzer, be sure to read the documentation of the corpus. This may include more specific information about the data and how to cite it. This page gives some general advice on citing documents from I-analyzer in your research.
+
+### Citing a document from the I-analyzer interface
+
+If you downloaded a document in I-analyzer, or read it online in the I-analyzer interface, your citation should make it clear that you retrieved it through here. This provides accountability: loading corpora into I-analyzer is a process that may introduce mistakes, so you should always cite the resource that you actually referenced.
+
+For example, a citation might look something like the following:
+
+> Doe, J. (1900, Jan 1). A very interesting article. *The Times.* http://ianalyzer.hum.uu.nl/document/fascinating-corpus/12345678
```

Comment:
How to deal with the fact that the links will not be accessible to anyone? I don't have an answer, just wonder whether we should include a caveat about this.

Reply:
Good point. Yes, I will add a comment about it - I don't think we can do much more than that. In the interest of FAIR data, it would be nice if I-analyzer corpora were public, but that is not always possible.

That said, before the current move to open access journals, you could expect that the vast majority of references in an article would not be publicly accessible, so I would say that citing a closed resource is not a problem per se.